src/pages/archivePage/ArchivePage.jsx

```diff
@@ -1,9 +1,50 @@
+import axios from "axios";
 import { Archive } from "components/index";
+import { useNotes } from "context";
+import { useEffect } from "react";
+import "./archivePage.css";
 
 function ArchivePage() {
+  const encodedToken = localStorage.getItem("token");
+  const {
+    noteState: { archivedNotes },
+    noteDispatch,
+  } = useNotes();
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = axios.get("/api/archives", {
+          headers: {
+            authorization: encodedToken,
+          },
+        });
+        if (response.status === 200) {
+          noteDispatch({
+            type: "SET_ARCHIVE",
+            payload: (await response).data.archives,
```

Comment:
suggestion - use await in the axios call

Reply:
Forgot to do it. Thanks for pointing out.